pkg/k8sutil/k8sutil.go

```diff
@@ -583,3 +584,43 @@ func (k *K8sutil) CreateDataNodeDeployment(deploymentType string, replicas *int3
 
 	return nil
 }
+
+func (k *K8sutil) CreateCerebroConfiguration(clusterName, cert string) map[string]string {
+	clusterName = fmt.Sprintf("https://%s:9200", fmt.Sprintf(fmt.Sprintf("elasticsearch-%s", clusterName)))
+
+	x := map[string]string{}
+	x["application.conf"] = fmt.Sprintf(`
+play.ws.ssl {
+        trustManager = {
+                stores = [
+				{ type = "PEM", path = "%s/cerebro.pem" },
```

Comment:
I think you're using the cert to talk to the ES cluster, please confirm, but makes sense.

Reply:
yes, that's the point